packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -103,6 +108,7 @@ void testWidgets(
       }
       tester._recordNumberOfSemanticsHandles();
       test_package.addTearDown(binding.postTest);
+      _AssetMocker.mock();
```

Comment:
This will run at the beginning of every test. I make it so it will only mock the first time it is called, after that will be no op

Reply:
I think it would be better to move this into the binding initialization code.

Reply:
done